### DISCUSSION TITLE
avrdude Wiki updates

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1381] [Announcements]
I have created the following simple FAQ pages:
https://github.com/avrdudes/avrdude/wiki/FAQ

This is chosen from the entries here.
https://github.com/mcuee/avrdude/wiki

It would be good to add more FAQ entries for the users.

**Comment by mcuee** [comment]
Wiki will be updated along the way.

**Comment by mcuee** [comment]
Updates on 2024-01-10.
New page created:
https://github.com/avrdudes/avrdude/wiki/Known-limitations-of-avrdude

Main page updated:
https://github.com/avrdudes/avrdude/wiki

**Reply by mcuee** [reply]
New Wiki page created for benchmark.
https://github.com/avrdudes/avrdude/wiki/avrdude-benchmark-for-Flash-and-EEPROM-programming

**Comment by mcuee** [comment]
Close this one for now.

**Reply by mcuee** [reply]
We can always add new Wiki page when needed.

**Comment by mcuee** [comment]
Two new pages have been created to document the build instructions for OpenBSD and NetBSD.

https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-OpenBSD
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-NetBSD

**Reply by stefanrueger** [reply]
Excellent stuff! Thanks

**Reply by mcuee** [reply]
@MCUdude 

Generic auto-tools build instructions added.
https://github.com/avrdudes/avrdude/wiki/General-Build-Instructions

I will add the instructions to all the platforms as well, like Linux/Windows MSys2/macOS/FreeBSD/OpenBSD/NetBSD.

**Comment by mcuee** [comment]
Minor updates to FAQ page.
https://github.com/avrdudes/avrdude/wiki/FAQ
